src/components/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from './Avatar';
-export * from './Badge';
+export * from '../legacy/Badge';
```

Comment:
Warum exportierst du die Badge hier noch einmal aus `../legacy`? Weiter unten legen wir ja schon mit `export * from './../legacy';` fest, dass alle Komponenten aus Legacy exportiert werden. An dieser Stelle exportieren wir ja die normalen Komponenten, daher sollte sich an dem Export eigentlich nichts ändern 🤔 
```suggestion
export * from './Badge';
```

Reply:
Hatte das vergessen zu pushen, kam vom automatischen anpassen der Imports.